src/main/scala/mesosphere/marathon/core/event/Events.scala

```diff
@@ -186,6 +186,13 @@ case class MesosStatusUpdateEvent(
   eventType: String = "status_update_event",
   timestamp: String = Timestamp.now().toString) extends MarathonEvent
 
+// TODO these must be removed when we have a distinct MarathonTaskStatus
```

Comment:
Exactly. I would actually like to merge his PR first because I think it is easier to rebase mine on top of his. The other way 'round shouldn't be a big problem, though.